README.md

```diff
@@ -1,4 +1,10 @@
-## Rock Paper Scissors: A Force-Move Games POC
+## Rock Paper Scissors: an example ForceMove game
+
+This app is an example of a game built on our [ForceMove](https://magmo.com/force-move-games.pdf) protocol. You can play against a friend or against our bots on the ropsten testnet.
+
+![splash](./screens.png 'screens')
+
+To run the app on your machine, clone down the code and follow the instructions below. 
```

Comment:
@geoknee Minor wording tweak. You would usually say "clone" or "pull down," but not "clone down."